salt/modules/win_network.py

```diff
@@ -289,3 +294,94 @@ def ip_addrs6(interface=None, include_loopback=False):
                                         include_loopback=include_loopback)
 
 ipaddrs6 = ip_addrs6
+
+def connect(host, port=None, **kwargs):
+    '''
+    Test connectivity to a host using a particular
+    port from the minion.
+
+    .. versionadded:: 2014.7
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' network.connect archlinux.org 80
+
+        salt '*' network.connect archlinux.org 80 timeout=3
+
+        salt '*' network.connect archlinux.org 80 timeout=3 family=ipv4
+
+        salt '*' network.connect google-public-dns-a.google.com port=53 proto=udp timeout=3
+    '''
+
+    ret = {'result': None,
+           'comment': ''}
+
+    if not host:
+        ret['result'] = False
+        ret['comment'] = 'Required argument, host, is missing.'
+        return ret
+
+    if not port:
+        ret['result'] = False
+        ret['comment'] = 'Required argument, port, is missing.'
+        return ret
+
+    proto = kwargs.get('proto', 'tcp')
```

Comment:
Generally we like to have explicit arguments instead of pulling them out of kwargs. Is there a reason they need to be in kwargs?